test/baselines/semi/test_recommend.py

```diff
@@ -116,6 +118,56 @@ def test_EMO_no_open_bracket_2(self):
         self.assertEqual(expect_EMO, result_EMO)
 
     def test_recommend_for_method(self):
-        result_emos = recommend_for_method(self._method)
+        result_emos = recommend_for_method('\n'.join(self._method))
         all_possible_emos = {(1, 4)}
         self.assertEqual(set(result_emos), all_possible_emos, str(result_emos))
+
```

Comment:
@KatGarmash  do we need that commented code?

Reply:
maybe you want this https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures